maple-diffusion/MapleDiffusion.swift

```diff
@@ -1,15 +1,16 @@
 import MetalPerformanceShadersGraph
 import Foundation
+import Accelerate
```

Comment:
This import is no longer needed, right? Other than that LGTM

Reply:
Oh, yeah, sorry, missed it:)